docker-compose/dev.yml

```diff
@@ -48,6 +49,17 @@ services:
           $$ADDITIONAL_ARGS
       "
 
+  poet-validator-registry:
+    image: hyperledger/sawtooth-poet-validator-registry-tp:1.0.1
+    env_file:
+      - ../.env
+    entrypoint: |
+      bash -c "
+        if [ \"$$REMME_CONSENSUS\" = \"poet-simulator\" ]; then
+          poet-validator-registry-tp -vv -C tcp://validator:4004
```

Comment:
What function does `poet-validator-registry-tp` perform? can it be reused for our own consensus for tracking validators list?

Reply:
Essentially it handles the list of validators for PoET. It cannot be reused because it uses validator/SGX specific functions.